modules/oe_content_publication/config/post_updates/00003_create_body_field/field.field.node.oe_publication.body.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.body
+    - node.type.oe_publication
+  module:
+    - text
+id: node.oe_publication.body
+field_name: body
+entity_type: node
+bundle: oe_publication
+label: Body
+description: ''
+required: false
+translatable: true
```

Comment:
`translatable: false`

Reply:
The task description at https://citnet.tech.ec.europa.eu/CITnet/jira/browse/D8TEWPPE-112 set the `translatable` property to `true`

Reply:
We set `translatable: true` in the field.storage.*.*.yml configurations.  `translatable: false` is in all field.field.*.*.*.yml files. See example   build/modules/contrib/oe_content/modules/oe_content_project/config/install/field.field.node.oe_project.body.yml